app/assets/stylesheets/application.css.scss

```diff
@@ -16,15 +16,33 @@
 
  */
 
+body {
+  background-color: white;
+}
+
+.name {
+  padding-top: 7px;
+  padding-right: 10px;
+}
+
 .review_body {
   max-width: 500px;
   word-wrap: break-word;
 }
 
+.top-bar {
+  background-color: #7EBAC2;
+}
+
+
+.top-bar-section li:not(.has-form) a:not(.button) {
+  background: #E29714;
+}
+
 .upvote {
-  color: red;
+  color: #E30922;
```

Comment:
Color literals like `#E30922` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E30922` should be written as `#e30922`